src/components/AddTransaction.js

```diff
@@ -1,8 +1,8 @@
-import React, {useState, useContext} from 'react'
-import { GlobalContext } from '../context/GlobalState';
+import React, { useState, useContext } from "react";
+import { GlobalContext } from "../context/GlobalState";
```

Comment:
There is no difference between Single or double quotes. The choice of quoting style is up to the programmer.
Code style must be consistent across all files with one type of quotes within JSX attributes.

> When working with JSX best practice is to use double quotes directly (no-braces) if just a simple string, or use backtick if interpolating a var into the string.